src/sentry/migrations/0028_discover_query_upgrade.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+# Generated by Hand
+from __future__ import unicode_literals, print_function
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+from sentry.discover.models import DiscoverSavedQuery, DiscoverSavedQueryProject
+from sentry.api.event_search import SearchVisitor
+from django.db import migrations
+
+# Aggregates are now fields
+def convert_field(fieldname, unique, reverse):
+    if fieldname == "count":
+        fieldname = u"count()"
+    elif unique:
+        fieldname = u"count_unique({})".format(fieldname) if unique else field
```

Comment:
You don't need the `if unique` here.

Reply:
🙃 the else was on an old variable name too -- good catch